Makefile

```diff
@@ -49,7 +49,11 @@ TEST_PREFIX ?= ${IDRIS2_CURDIR}/build/env
 IDRIS2_BOOT_PREFIX := ${IDRIS2_CURDIR}/bootstrap-build
 
 # These are the library path in the build dir to be used during build
-export IDRIS2_BOOT_PATH := "${IDRIS2_CURDIR}/libs/prelude/build/ttc${SEP}${IDRIS2_CURDIR}/libs/base/build/ttc${SEP}${IDRIS2_CURDIR}/libs/linear/build/ttc${SEP}${IDRIS2_CURDIR}/libs/network/build/ttc${SEP}${IDRIS2_CURDIR}/libs/contrib/build/ttc${SEP}${IDRIS2_CURDIR}/libs/test/build/ttc"
+IDRIS2_LIBRARIES = prelude base linear network contrib test
```

Comment:
I think this approach is now actively going against #2677

Reply:
Both lines have the same set of libraries.  Or am I missing something?

Reply:
As long as it's not installing `papers` it's fine, no? I recently forgot to install `linear` as part of a `make install-libs` and it caused a bunch of problems...
CI seems happy as well?  : )

Reply:
Could someone please, point to a line where `papers` gets installed?  I went through the changes one more time, and I just can't see it.

The only change regarding `papers` is building (but not installing) documentation: https://github.com/ska80/Idris2-SH/blob/9ebfa2e0509b984d673b3d2c4eb736a4ac2dfff1/Makefile#L109

If this PR is pointless, let's close it :)

Reply:
@ska80, afaict `papers` never gets installed as part of this PR, which is why I'm mostly happy to merge it  ^^

The only concern I have is whether `papers` should be removed from line 109, since the docs are not installed as part of the `install-libdocs` target; they're built but never used. (Alternatively: should they be added to the `install-libdocs` target? My gut instinct is "No", with the reasoning being that if you want to use the `papers` library, you have to do everything yourself, but that might not be the best approach...)

Reply:
@CodingCellist Ok, I have removed the line that builds docs for `papers`.  Thanks for reviewing!